src/main/java/org/springframework/data/repository/util/QueryExecutionConverters.java

```diff
@@ -152,15 +151,6 @@
 
 			ALLOWED_PAGEABLE_TYPES.add(io.vavr.collection.Seq.class);
 		}
-
-		if (ReactiveWrappers.isAvailable()) {
```

Comment:
Care to drop a note why this can be removed?

Reply:
This change is part of the commit https://github.com/spring-projects/spring-data-commons/pull/415/commits/a32a7b067c5c97fb97a3f0f54dc9c2191b008f98 explaining that we use Spring's `ReactiveAdapterRegistry` now.